project/Dependencies.scala

```diff
@@ -44,7 +44,7 @@ object Dependencies {
     val badrows            = "1.0.0"
     val igluClient         = "0.6.2"
     val igluCore           = "0.5.0"
-    val processingManifest = "0.1.0"
+    val processingManifest = "0.2.0"
```

Comment:
I don't think it is used at the moment. Either way, processing manifest should be removed.